deployments/kwok-perf-test/affinity.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RANDOM=0
+OUTPUT_PATH="./output/affinity.yaml"
+NUM_PODS=0
+NUM_NODES=0
+OPERATORS=("In" "NotIn")
+OPERATORS_STATE=0
+j=0
+
+show_help() {
+  cat << EOF
+Invalid option: -$OPTARG
+Usage: $0 <pod_count> <node_count>
+
+Options:
+  -o,             Specifies the location of the output yaml file (default is ./output/affinity.yaml)
+
+Arguments:
+  <pod_count>     Number of pod to create (required).
+  <node_count>    Number of kwok nodes (required).
+EOF
+}
+
+# Process command-line options
+while getopts ":o:" opt; do
+  case $opt in
+    o)
+      OUTPUT_PATH=$OPTARG
+      ;;
+    \?)
+      show_help
+      exit 1
+      ;;
+    :)
+      show_help
+      exit 1
+      ;;
+  esac
+done
+
+# Shift the processed options out of the command-line arguments
+shift $((OPTIND-1))
+
+# Check if pod count and node count are provided
+if [ $# -ne 2 ]; then
+  show_help
+  exit 1
+fi
+
+NUM_PODS=$1
+NUM_NODES=$2
```

Comment:
What's the minimum number of pods/nodes that make sense? Looking at the code below, we need at least 4 nodes. We need to validate them and show an error if it's necessary.